src/CommonAssemblyInfo.cs

```diff
@@ -18,16 +15,3 @@
 #endif
 [assembly: NeutralResourcesLanguage("en-US")]
 [assembly: AssemblyMetadata("Serviceable", "True")]
-
-// ===========================================================================
-//  DO NOT EDIT OR REMOVE ANYTHING BELOW THIS COMMENT.
-//  Version numbers are automatically generated based on regular expressions.
-// ===========================================================================
-
-#if ASPNETODATA
-#if !BUILD_GENERATED_VERSION
-[assembly: AssemblyVersion("5.9.0.0")] // ASPNETODATA
```

Comment:
WebStack.versions.settings.targets generates a cs assembly info file from the data at the top of that file. The cs file is dropped into the obj dir during compilation. When compiling in VS, you get the same behavior as the build system so the version is whatever is in WebStack.versions.settings.targets.